tests/default_vars.sh

```diff
@@ -9,6 +9,8 @@
 
   THRD=1
 
+  TASKS_atmaero=198 ; INPES_atmaero=4 ; JNPES_atmaero=8; WPG_atmaero=6
```

Comment:
Sounds like parsing issue on jet: hafs_regional_docn and hafs_regional_docn_oisst. We need to remove the space before semi colons (TASKS_atmaero=198; INPES_atmaero=4;). With the fix, all tests run ok on jet.

Reply:
@jkbk2004 Good catch!